apis/python/src/tiledbsoma/collection.py

```diff
@@ -109,6 +110,25 @@ def _create(self, soma_type: str) -> "CollectionBase[CollectionElementType]":
         self._cached_values = {}
         return self
 
+    # lazy tiledb.Group handle for reuse while self is open
+    _open_tiledb_group: Optional[tiledb.Group] = None
+
+    @property
+    def _tiledb_obj(self) -> tiledb.Group:
```

Comment:
same question as with tiledb_arra.py::_tiledb_obj -- why is this lazy?   I'd rather have the early error reporting and latency win from early....

Reply:
Made them eager.